src/indexable.cr

```diff
@@ -211,6 +211,16 @@ module Indexable(T)
     size == 0
   end
 
+  # Optimized version of `Indexable#equals?` used when `other` is also an
+  # `Indexable`.
+  def equals?(other : Indexable(T))
```

Comment:
Ah, didn't know. Thanks.

Do you know how to get the docs to link to the correct overload? Couldn't figure it out.